third_party/libunwindstack/Unwinder.cpp

```diff
@@ -211,6 +233,16 @@ void Unwinder::Unwind(const std::vector<std::string>* initial_map_names_to_skip,
       // Once a frame is added, stop skipping frames.
       initial_map_names_to_skip = nullptr;
     }
+    if (map_info != nullptr && ShouldStop(functions_to_stop_at, cur_pc)) {
```

Comment:
I would add an empty line before and after this block.

Reply:
Done.